gis/qgis3/Portfile

```diff
@@ -170,7 +169,7 @@ configure.args-append \
                     -DQWT_INCLUDE_DIR=${prefix}/libexec/qt5/lib/qwt.framework/Versions/Current/Headers \
                     -DQWT_LIBRARY=${prefix}/libexec/qt5/lib/qwt.framework/Versions/Current/qwt \
                     -DSPATIALINDEX_LIBRARY=${prefix}/lib/libspatialindex.dylib \
-                    -DUSE_OPENCL=OFF \
+                    -DUSE_OPENCL=ON \
```

Comment:
it doesn't look like your declaring a dependency on a port -  is this doing what you want it to do?

Reply:
It will build with system OpenCL.

Reply:
> it doesn't look like your declaring a dependency on a port - is this doing what you want it to do?

Yes.